cardano_node_tests/tests/test_pools.py

```diff
@@ -531,6 +635,7 @@ def test_create_stake_pool(
             request=request,
         )
 
+    @pytest.mark.dbsync
```

Comment:
For now, I added those checks only to one test - I can add them to more, not sure if there are any rules about making `db-sync` checks.